src/rpc/rpc.cc

```diff
@@ -1211,6 +1212,16 @@ future<> server::connection::send_unknown_verb_reply(std::optional<rpc_clock_typ
       return isolation_config{};
   }
 
+  void server::warn_stuck_handler() {
+      sstring ids = "";
+      _proto->with_handlers([&ids] (auto msg_id, auto& handler) {
+          if (handler.use_gate.get_count() != 0) {
+              ids += fmt::format("{}", msg_id);
```

Comment:
Don't you want to put comma between numbers?

Reply:
better have `ids` be a `std::vector<msg_id>` and print it with `"{{{}}}", fmt::join(ids, ", ").